src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidation.java

```diff
@@ -33,10 +36,20 @@ public void validate(CompanyProfileApi companyProfileApi) throws EligibilityExce
             return;
         }
         var officers = officerService.getOfficers(companyProfileApi.getCompanyNumber());
-        var officerCheck = isOfficerDirector(officers.getItems(), officers.getActiveCount());
-        if (!officerCheck) {
-            ApiLogger.info(String.format("Company Officers validation failed for: %s", companyProfileApi.getCompanyNumber()));
-            throw new EligibilityException(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS);
+        var activeOfficersCount = officers.getActiveCount();
+        ApiLogger.debug(String.format("Company has %s active officers", activeOfficersCount));
+
+        if(!multipleOfficerJourneyFlag) {
+            var officerCheck = isOfficerDirector(officers.getItems(), activeOfficersCount);
+            if(!officerCheck) {
+                ApiLogger.info(String.format("Company Officers validation failed for: %s", companyProfileApi.getCompanyNumber()));
+                throw new EligibilityException(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS);
+            }
+        } else {
+            if(activeOfficersCount != null && activeOfficersCount > 5) {
+                ApiLogger.info(String.format("Company Officers validation failed for: %s", companyProfileApi.getCompanyNumber()));
```

Comment:
i would add the officer count limit into the message so we know why it failed eg.
String.format("Company Officers validation failed for: %s more than 5 active officers", companyProfileApi.getCompanyNumber())

Reply:
the original validation checked if the company officer was a director, the feature flagged 5 officers one doesn't check if they are directors. Do we need to also make sure the multiple officers are also directors?

Reply:
In case of multiple officers, they don't have to be directors. The active director is required if there's only one active officer.